tests/assembly/ignore-inline-never.rs

```diff
@@ -21,6 +21,6 @@ fn actually_inlined(a: u64) -> u64 {
 pub extern "C" fn fun(a: u64) -> u64 {
     // CHECK-LABEL: fun:
     actually_inlined(a)
-    // CHECK-NEXT: r{{[0-9]}} = r{{[0-9]}}
```

Comment:
Can you explain this change in a commit message?

Reply:
Will do